### PR TITLE
Fixes #7282: Add info about policy and inventory dates

### PR DESCRIPTION
**Pull request by matya**
Show info about when was the last time the inventory was sent
and the policy was updated when running rudder agent info.

### CODE REVIEW

**Comment by matya** [comment]
@peckpeck Please also consider this.

File: share/commands/agent-info [hunk 1]
```diff
@@ -17,6 +17,12 @@ HASH=$(echo "${HASH}" | sed -e 's/.*= //')
 POLICY=$(cat /var/rudder/cfengine-community/policy_server.dat 2>/dev/null)
 [ $? -ne 0 ] && POLICY="Not yet configured"
 
+POLICY_FETCHED=$( stat -c %y /var/rudder/cfengine-community/last_successful_inputs_update 2> /dev/null )
+[ $? -ne 0 ] && POLICY_FETCHED="Not updated yet" || POLICY_FETCHED=$( cut -d. -f1 <<<"$POLICY_FETCHED"  )
```
Comment (peckpeck) [inline share/commands/agent-info]: We prefer the form "${POLICY_FETCHED}" for variables.
And '<<<' is a bashism, you should use 'echo |' instead

**Comment by matya** [comment]
@peckpeck , habits to evade subshells :-)
reworked the RP to respect this and also match the message wording.

**Comment by peckpeck** [comment]
Hehe!
Good thanks.

**Pull request merged**